openslides/topics/models.py

```diff
@@ -17,19 +18,26 @@ class Topic(RESTModelMixin, models.Model):
     text = models.TextField(blank=True)
     attachments = models.ManyToManyField(Mediafile, blank=True)
 
+    # In theory there could be one then more agenda_item. But support only one.
+    # See the property agenda_item.
+    agenda_items = GenericRelation(Item)
+
     class Meta:
         default_permissions = ()
 
     def __str__(self):
         return self.title
 
+    @classmethod
+    def all_query(cls):
+        return cls.objects.prefetch_related('attachments', 'agenda_items')
+
```

Comment:
No. It did not work. I also read the documentation. Therefore I think it is a django bug that Managers on Mixins are not supported.

Currently I override the objects manager but with a different method ```get_full_queryset()```.

Please have a look at my newest commit. I added tests for each RestModel viewset that counts the queries used for a list view. Currently it is a disaster :) I fixt some cases but there are still much more queries then needed. Maybe you can also have a look at it? This has nothing to do with caching. In this tests caching is disabled (as usual in tests).